### DISCUSSION TITLE
Prevent auto-execution of notebooks

### DISCUSSION THREAD

**Discussion by kbkartik** [fonsp/Pluto.jl#968] [Questions about using Pluto]
I am using Julia (v1.5.3) with Pluto (v0.12.21) and PlutoUI (v0.7.2). How can I disable auto-execution of cells when a new instance of the notebook is spawned?

**Comment by fonsp** [comment]
#259

**Reply by kbkartik** [reply]
I did try the solution you mentioned in the above issue: `withenv(Pluto.run, "run_notebook_on_load" => false)`. However, it doesn't help and executes all cells on startup.

**Reply by fonsp** [reply]
I just edited my comment to fix the outdated syntax, can you try again?

**Reply by kbkartik** [reply]
The updated syntax works. Thanks!
